dbt/adapters/duckdb/utils.py

```diff
@@ -75,7 +76,7 @@ def as_dict(self) -> Dict[str, Any]:
 class TargetConfig:
     relation: BaseRelation
     column_list: Sequence[Column]
-    config: RuntimeConfigObject
+    config: Any  # TODO
```

Comment:
This is probably the safest since our usage of this config object is pretty much limited to `get` methods